ReSharper.FSharp/test/data/features/quickFixes/replaceReturnType/Replace type with attribute.fs

```diff
@@ -0,0 +1,4 @@
+type FooAttribute() =
+    inherit System.Attribute()
```

Comment:
Minor: maybe it could use an existing attribute instead of defining one to keep the test simpler.